app/src/main/java/com/github/se/project/ui/overview/Overview.kt

```diff
@@ -0,0 +1,165 @@
+import android.content.Context
+import android.util.Log
+import android.widget.Toast
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountBox
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import com.github.se.project.model.lesson.Lesson
+import com.github.se.project.model.lesson.LessonViewModel
+import com.github.se.project.model.profile.ListProfilesViewModel
+import com.github.se.project.model.profile.Profile
+import com.github.se.project.model.profile.Role
+import com.github.se.project.ui.navigation.BottomNavigationMenu
+import com.github.se.project.ui.navigation.LIST_TOP_LEVEL_DESTINATIONS_STUDENT
+import com.github.se.project.ui.navigation.LIST_TOP_LEVEL_DESTINATIONS_TUTOR
+import com.github.se.project.ui.navigation.NavigationActions
+import com.github.se.project.ui.navigation.Screen
+
+@Composable
+fun HomeScreen(
+    listProfilViewModele: ListProfilesViewModel,
+    lessonViewModel: LessonViewModel,
+    navigationActions: NavigationActions,
+) {
+
+  val context = LocalContext.current
+  val currentProfile = listProfilViewModele.currentProfile.collectAsState().value
+  val lessons = lessonViewModel.currentUserLessons.collectAsState().value
+
+  // lessonViewModel.getLessonsByUserId(currentProfile?.uid ?: "")
+  when (currentProfile?.role) {
+    Role.TUTOR -> lessonViewModel.getLessonsForTutor(currentProfile.uid)
+    Role.STUDENT -> lessonViewModel.getLessonsForStudent(currentProfile.uid)
+    Role.UNKNOWN -> Toast.makeText(context, "Unknown Profil", Toast.LENGTH_SHORT).show()
+    null -> Toast.makeText(context, "null Profil", Toast.LENGTH_SHORT).show()
+  }
+
+  // Determine which bottom navigation items to show based on the user's role
+  val LIST_TOP_LEVEL_DESTINATIONS =
+      when (currentProfile?.role) {
+        Role.TUTOR -> LIST_TOP_LEVEL_DESTINATIONS_TUTOR
+        Role.STUDENT -> LIST_TOP_LEVEL_DESTINATIONS_STUDENT
+        Role.UNKNOWN -> LIST_TOP_LEVEL_DESTINATIONS_STUDENT
```

Comment:
maybe in that "UNKNOWN" case (that should never happen) we could use an empty list and show a toast, but this is not mandatory for the moment !